lib/Server.js

```diff
@@ -441,8 +442,11 @@ Server.prototype.checkHost = function(headers) {
 	const idx = hostHeader.indexOf(":");
 	const hostname = idx >= 0 ? hostHeader.substr(0, idx) : hostHeader;
 
+	// always allow requests with explicit IP-address
+	if(ip.isV4Format(hostname)) return true;
```

Comment:
Should I do another PR for this? Did anyone ever use IPv6 in such a context? 
Using `http://[::1]:1234/` is absolutely possible, but that will already break in the lines before as a colon no longer works to split out the port-number from the host-header.